### ISSUE TITLE
Unit test

### ISSUE THREAD

**Issue by MikeGertrudes** [pricelinelabs/omni-slider#6]
Worth creating a unit test for this module? Could be valuable as contributors begin to refactor and add features.

**Comment by chrisrng** [comment]
Yeah for sure we need a battery of unit tests. Especially for IE and mobile compatibility.

**Comment by bcVamp** [comment]
Yes, we need unit test workflow to establish TDD/BDD development in this project. What unit test toolsets are you guys considering?

**Comment by jjmasse** [comment]
Might be worth trying out CodeClimate, it is free for all open source projects.
https://codeclimate.com/oss
It also integrates very well with GitHub and pull requests.
